python/tests/plugins/validators/test_validators.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from aac.plugins.validators import get_validation_definition_from_plugin_definitions
+
+
+class TestValidationModule(TestCase):
+    def test_get_validation_definition_from_plugin_definitions(self):
+        parsed_definition = get_validation_definition_from_plugin_definitions(__name__, TEST_VALIDATION_DEFINITION_STRING)
+        definition = parsed_definition.get("validation")
+        definition_name = definition.get("name")
+        self.assertEqual(TEST_VALIDATION_NAME, definition_name)
```

Comment:
Is it currently the case that validation functions are not being used? If they are, how are they being defined?